src/Plane/PlaneFileGlue.cpp

```diff
@@ -208,3 +210,21 @@ PlaneGlue::WriteFile(const Plane &plane, Path path)
   buffered.Flush();
   file.Commit();
 }
+
+StaticString<42>
+PlaneGlue::GetFileName(const Plane &plane)
+{
+  StaticString<42> filename(plane.registration);
+  filename += _T(".xcp");
+  return(filename);
+}
+
+// Simplified interface to write plane file including file extension handling
+void
+PlaneGlue::WriteFile(const Plane &plane)
+{
+  StaticString<42> filename( GetFileName(plane) );
+  const auto path = LocalPath( filename );
+  LogFormat(_T("WriteFile path:%s"), path.c_str() );
```

Comment:
Useless logging call

Reply:
Used during test, will be removed in PR for plane masses